### ISSUE TITLE
Minute hand sometimes doesn't tick over when second hand reaches 12

### ISSUE THREAD

**Issue by gorbypark** [mateoguzmana/react-native-clocks#5]
**Describe the bug**
Sometimes the minute hand is out of sync with the seconds hand.  I haven't been able to reproduce exactly how to trigger it, but sometimes the minute hand ticks to the next minute at random spots.  If the app is reloaded, it will tick over at another random spot.  Also, occasionally it does tick at 12 like it should.

**To Reproduce**
1. use clock component `<Clock />`

**Expected behavior**
I'd think the minute hand would tick when the seconds hand reaches 12

**Screenshots**
Here's a screen grab of it ticking over at about 34 seconds.
https://github.com/mateoguzmana/react-native-clocks/assets/19569469/7906b865-41a3-4a42-a348-5bb75660f86d

**Comment by gorbypark** [comment]
Had a quick look at the code and it seems the issue is that the increaseMinutes function is setting an interval of one minute, so the minutes will tick over one minute after the interval is created.  It also is likely that the hour hand will exhibit the same issue.

I think there is really two options,
1) increaseMinutes (or hours)  should set an interval only until the top of the minute (or hour).  It should also set a timeout when first mounted that takes in account of how long it is until the top of the minute (or hour), and then sets an interval for one minute only then.

2) There should only be one interval running once per second, with some logic that increments the minutes and hours at the appropriate time.

This seems to be the issue, it is run when the component mounts, so the minute hand will tick over one minute past mount.
```js
const increaseMinutes = setInterval(() => {
  setCurrentMinutes((val) => val + 1);
}, ONE_MINUTE_IN_MS);
    ```

**Comment by mateoguzmana** [comment]
Fixed by #6
